superset-frontend/src/components/Menu/SubMenu.tsx

```diff
@@ -158,7 +158,6 @@ const SubMenuComponent: React.FunctionComponent<SubMenuProps> = props => {
   const [navRightStyle, setNavRightStyle] = useState('nav-right');
   const [navRightCol, setNavRightCol] = useState(8);
 
-  const { headerSize = 2 } = props;
```

Comment:
I think that we can remove headerSize from SubMenuProps (line 153), it doesn't seem to be used anymore.

Reply:
We can also remove passing `headerSize` to SubMenu from `CssTemplatesList` and `AnnotationLayersList`